fmriprep/workflows/epi.py

```diff
@@ -138,6 +138,17 @@ def ref_epi_t1_registration(reportlet_suffix, inv_ds_suffix, name='ref_epi_t1_re
 
     explicit_mask_epi = pe.Node(fsl.ApplyMask(), name="explicit_mask_epi")
 
+    if settings['freesurfer']:
+        bbregister = pe.Node(
+            BBRegisterRPT(
+                contrast_type='t2',
+                init='header',
+                registered_file=True,
+                out_fsl_file=True,
```

Comment:
Ugh. `remove_unnecessary_outputs` is `True` by default.

It does seem like if I say "Make this file", it should at least throw a warning if it's unused, but I suppose setting a config variable is easy enough.